backend/src/graphql/resolver/UserResolver.test.ts

```diff
@@ -1558,11 +1651,33 @@ describe('UserResolver', () => {
           resetToken()
         })
 
+        it('returns user with new moderator-role', async () => {
```

Comment:
What is new compared to line 1693?

Reply:
this test compares setting a user with admin role the role moderator and
the test beginning at line 1693 compare setting a user with admin role the role admin